main.nf

```diff
@@ -153,6 +165,7 @@ params.pmd_udg_type = 'half'
 params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
 params.email = false
 params.plaintext_email = false
+params.pop_gen_bedfile = "$baseDir/1240K.pos.list_hs37d5.0based.bed"
```

Comment:
Can we bundle this for legal reasons with the pipeline? If yes, nice - if not, we should provide it via a parameter.